src/components/alert/alert.js

```diff
@@ -59,6 +69,27 @@ class Alert extends Dialog {
       'data-role': props['data-role']
     };
   }
+
+  /**
+   * Handles keyboard focus leaving the dialog
+   * element.
+   *
+   * Assumes that, if no close icon is displayed,
+   * no other element can receive keyboard focus.
+   * Therefore focus should remain on the dialog
+   * element while it is open.
+   *
+   * @override
+   * @return {Void}
+   */
+  onDialogBlur(ev) {
```

Comment:
It shouldn't. When I tested it the value of `enableBackgroundUI` didn't have any effect on `onDialogBlur` i.e. focus remained in the dialog regardless.